setup.py

```diff
@@ -458,6 +452,11 @@ def _read_requirements(filename: str) -> List[str]:
     python_requires=">=3.8",
     install_requires=get_requirements(),
     ext_modules=ext_modules,
+    # UPSTREAM SYNC: needed for sparsity
```

Comment:
where is "version" being set?

Reply:
The version is getting from get_vllm_version() function which reads from __init__.py